src/app/data-planes/views/DataPlanePolicySummaryView.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <RouteView
+    v-slot="{ route, t }"
+    name="data-plane-policy-summary-view"
+    :params="{
+      mesh: '',
+      policyPath: '',
+      policy: '',
+      codeSearch: '',
+      codeFilter: false,
+      codeRegExp: false,
+    }"
+  >
+    <DataSource
+      v-slot="{ data, error }: PolicySource"
+      :src="`/meshes/${route.params.mesh}/policy-path/${route.params.policyPath}/policy/${route.params.policy}`"
+    >
+      <AppView>
+        <template #title>
+          <PolicySummaryTitle
+            :title="t('policies.routes.item.title', { name: route.params.policy })"
+            :link-to="{
+              name: 'policy-detail-view',
+              params: {
+                mesh: route.params.mesh,
+                policyPath: route.params.policyPath,
+                policy: route.params.policy,
+              },
+            }"
+          />
```

Comment:
We usually lean towards avoiding creating components for this sort of thing. Reason being, its far easier to see what's happening if I just see the same that is in every other route I know its fine, whereas now I'm wondering "what is it extra that PolicySummaryTitle is doing 🤔 ", even though its doing nothing extra. Moreover its not really saving us from writing less code, which I think could have been the reasoning here.

i.e. sometimes copy/paste is fine - in fact sometimes its best.

I'd much rather see the h2 and application standard `RouteTitle` here, even tho that means a little bit of repetition.

Reply:
I fully agree 👍 though the main intention was consistency in terms for style and layout, I see that the benefit for this component was not enough to reason the abstraction. Removed it and brought back the `h2` and `RouterLink` to the respective policy views.